SECURITY.md

```diff
@@ -0,0 +1,83 @@
+## Configuring Security
+
+
+The following variables configure container security using the socialLogin-1.0 feature.  
+
+### Configuration needed at image build time:
+
+ * The environment variable `sec_sso_providers` must be defined and contain a space delimited list of the identity providers to use. If more than one is specified, the user will be able to choose which one to authenticate with. Valid values are any of `oidc oauth facebook twitter github google linkedin`.  Specify `ARG sec_sso_providers="(your choice goes here)"` in your Dockerfile.
```

Comment:
usually the convention is to use all caps for env var names.

Reply:
fixed, what was I thinking?